README.md

```diff
@@ -82,11 +82,11 @@ This section should be referred to for next steps when any indicator reports a f
 | S0011        | Determines if Pe-postgres Service is Running and Enabled on relevant components    | Check the Service can be started and enabled   `puppet resource service pe-postgres ensure=running`  examing /var/log/puppetlabs/postgresql/<postgresversion>/postgresql-<today>.log for failures                                                                                                                                                                                                                                                                           | Raise a support case quoting reference S0011 along with the log below, showing an unsuccessful startup /var/log/puppetlabs/postgresql/<postgresversion>/ postgresql-<today>.log                                                       |
 | S0012        | Determines if Puppet produced a report within the last run interval                | https://puppet.com/docs/pe/2021.4/run_puppet_on_nodes.html#troubleshooting_puppet_run_failures                                                                                                                                                                                                                                                                                                                                                                              | Raise a support case quoting reference S0012 along with the output of `puppet agent -td > debug.log 2>&1`                                                                                                                             |
 | S0013        | Determines if a catalog successfully applied on Puppet Agent Last run              | https://puppet.com/docs/pe/2021.4/run_puppet_on_nodes.html#troubleshooting_puppet_run_failures                                                                                                                                                                                                                                                                                                                                                                              | Raise a support case quoting reference S0013 along with the output of  `puppet agent -td > debug.log 2>&1`                                                                                                                            |
+| S0014        | Making sure nothing in the command queue is older than 30 mins | Run `puppet agent -t` and observe if the Puppet run successfully finishes and starts a new one automatically within its run period. | Raise a support case quoting reference S0014 along with the following logs: `/var/log/puppetlabs/puppetdb/puppetdb.log` and a copy of `puppet agent -td > debug.log 2>&1`. Metrics can be used to pinpoint the issue and plotting a graph over time: https://support.puppet.com/hc/en-us/articles/231751308 |
```

Comment:
```suggestion
| S0014        | Indicates nothing in the command queue is older than a Puppet Run Interval | This can indicate that the PuppetDB performance is inadequate for the incoming requests. Review the PuppetDB Performance. Metrics can be used to pinpoint the issue https://support.puppet.com/hc/en-us/articles/231751308 | Raise a support case quoting reference S0014 along with the output of the Support Script  |
```